### PR TITLE
update peer dep version reference of graphql from 0.11 to 0.12.3

### PR DESCRIPTION
**Pull request by zebralight**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,6 +92,6 @@
     "supertest": "3.0.0"
   },
   "peerDependencies": {
-    "graphql": "^0.10.0 || ^0.11.0"
+    "graphql": "^0.12.3 || ^0.10.0 || ^0.11.0"
```
Comment (IvanGoncharov) [inline package.json]: It should be `^0.12.0` because it will include all `0.12.x` versions

**Comment by magic890** [comment]
@SimonDegraeve when is going to be released the package on npm registry with this fix?

**Comment by SimonDegraeve** [comment]
@magic890 I am not involved in this repository. Sorry.

**Comment by magic890** [comment]
@IvanGoncharov @wincent any on this update?

**Comment by IvanGoncharov** [comment]
@magic890 This PR both specify an incorrect version and failing test but #411 fixes all those issues.
As for progress overall, I don't have commit rights to this repo and `@wincent` moved to another project so I don't know if he would be able to review #411. 

I think I already pinged the right person in #411 so it should be merged in a few days.

@zebralight  Can you please close this PR in favor of #411 since it will help to solve confusion?

**Comment by IvanGoncharov** [comment]
@zebralight I merged #411 so I closing this PR.

@magic890 I already added both `0.12.x` and `0.13.x` as peer dependences and preparing a new release of NPM package.

**Comment by IvanGoncharov** [comment]
@zebralight @magic890 @SimonDegraeve `0.6.12` 📦 released with support for graphql `v0.12.x` and `v0.13.x`.

**Pull request closed without merging**